src/python/pppStartPipeline.py

```diff
@@ -28,10 +28,10 @@
     try:
         import ppp  # in case of running this module in the build folder
     except ImportError:
-        print("parallel-preprocessor python module `ppp` is not installed/importable")
+        # print("parallel-preprocessor python module `ppp` is not installed/importable")
         # print("if not installed, ppp module must be located in build folder")
         # print("../lib/ related to this script")
-        print("start to run `geomPipeline config.json`in an external process")
+        # print("start to run `pppGeomPipeline config.json`in an external process")
```

Comment:
What are you keeping this commented out code for?

Reply:
I found it was noisy,  when run  geomPipeline.py -h,   comment out for the moment,  not yet decided , whether change to multiple line comments or print to user